src/Workspaces/CSharp/Portable/Extensions/ITypeSymbolExtensions.TypeSyntaxGeneratorVisitor.cs

```diff
@@ -145,6 +147,21 @@ public TypeSyntax CreateSimpleTypeSyntax(INamedTypeSymbol symbol)
                     SyntaxFactory.TypeArgumentList(SyntaxFactory.SeparatedList(typeArguments)));
             }
 
+            private TupleTypeSyntax CreateTupleTypeSyntax(ITupleTypeSymbol symbol)
+            {
+                var list = new SeparatedSyntaxList<TupleElementSyntax>();
+                ImmutableArray<ITypeSymbol> types = symbol.Types;
```

Comment:
Yup -- much of the code base outside of the compiler, including workspaces and IDE, uses var. Different teams. Different religions. :smile: